modules/api-svc/package.json

```diff
@@ -63,19 +63,19 @@
   },
   "dependencies": {
     "@koa/cors": "^4.0.0",
-    "@mojaloop/api-snippets": "17.2.0",
-    "@mojaloop/central-services-error-handling": "^12.0.5",
-    "@mojaloop/central-services-logger": "^11.2.0",
-    "@mojaloop/central-services-metrics": "^12.0.5",
-    "@mojaloop/central-services-shared": "17.5.1",
-    "@mojaloop/event-sdk": "^11.0.2",
+    "@mojaloop/api-snippets": "17.2.5",
+    "@mojaloop/central-services-error-handling": "^12.0.7",
+    "@mojaloop/central-services-logger": "^11.2.2",
+    "@mojaloop/central-services-metrics": "^12.0.8",
+    "@mojaloop/central-services-shared": "18.1.2",
+    "@mojaloop/event-sdk": "^13.0.0",
     "@mojaloop/sdk-scheme-adapter-private-shared-lib": "workspace:^",
-    "@mojaloop/sdk-standard-components": "^17.1.1",
+    "@mojaloop/sdk-standard-components": "github:mojaloop/sdk-standard-components#feat/IPROD-93",
```

Comment:
will be updated to `"@mojaloop/sdk-standard-components": "^17.1.4"",` after that PR is merged:
https://github.com/mojaloop/sdk-standard-components/pull/189/files#